nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/FlowController.java

```diff
@@ -4919,6 +4925,22 @@ private void updateRemoteProcessGroups() {
         return new ArrayList<>(provenanceRepository.getEvents(firstEventId, maxRecords));
     }
 
+    public AuthorizationResult checkConnectableAuthorization(final String componentId) {
```

Comment:
I don't believe this is called.

Reply:
Correct. This was moved to ControllerFacade.java. I will remove it from FlowController.java.